api/src/pcapi/routes/pro/venue_providers.py

```diff
@@ -51,9 +51,10 @@ def create_venue_provider(body: PostVenueProviderBody) -> VenueProviderResponse:
 
     try:
         new_venue_provider = api.create_venue_provider(
-            body.providerId,
-            body.venueId,
-            VenueProviderCreationPayload(
+            provider_id=body.providerId,
+            venue_id=body.venueId,
+            author=current_user,
```

Comment:
```suggestion
            author=current_user.real_user,
```

sinon tu vas avoir des problemes avec le connect-as